ospfd/ospf_zebra.c

```diff
@@ -352,12 +408,18 @@ ospf_zebra_add (struct prefix_ipv4 *p, struct ospf_route *or)
   struct stream *s;
   struct ospf_path *path;
   struct listnode *node;
-  struct ospf *ospf = ospf_lookup ();
+
+  if (!ospf)
+    {
+      if (OSPF_DEBUG_TRACE)
```

Comment:
The purpose of the trace is for internal daemon activity. Please check new patch.